CHANGELOG.md

```diff
@@ -22,6 +22,10 @@
     - Rainbows
     - Sunspot
 
+- **Dropped method: `NewRelic::Agent.disable_transaction_tracing`**
+
+  Support for `NewRelic::Agent.disable_transaction_tracing` has been dropped. Users are encouraged to use `NewRelic::Agent.disable_all_tracing` or `NewRelic::Agent.ignore_transaction`.
```

Comment:
To me the "Support for" verbiage reads as if we have deprecated the method instead of completely ripping it out. Perhaps:

```markdown
The previously deprecated `NewRelic::Agent.disable_transaction_tracing` method has been removed. Users are...
```
```

Reply:
I like it!

Reply:
CHANGELOG [updated](https://github.com/newrelic/newrelic-ruby-agent/pull/1792/commits/bdf1a85c3cd353e1067b7b77a456f702aa07d6aa).